storage/src/archive/benches/utils.rs

```diff
@@ -29,7 +34,7 @@ pub type ArchiveType = Archive<TwoCap, Context, Key, (), Val>;
 /// The caller is responsible for closing or destroying it.
 pub async fn get_archive(ctx: Context, compression: Option<u8>) -> ArchiveType {
     let cfg = Config {
-        partition: PARTITION.into(),
+        partition: get_partition_name(),
```

Comment:
This change doesn't actually fix the issue. We need to update the `tokio` config here (the `PARTITION` used in archive/journal can stay unmodified 👍 ).

This is what we need to set: https://github.com/commonwarexyz/monorepo/blob/0d0b81090c85c887f7a1f3c2435e2c2f6e91ec2a/runtime/src/tokio/runtime.rs#L145

Reply:
Downside: you need to redo some of your changes. Positive: you can drop all the open questions about merging partitions.

Reply:
🤦 woops! Will do.